From ed867c8a032b53602573fcf04f0d39d7edb9a92e Mon Sep 17 00:00:00 2001
From: Nils Olof Paulsson <nilpa76@liu.se>
Date: Thu, 3 Aug 2017 16:08:42 +0200
Subject: [PATCH] no_utimensat

---
 file.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/file.c b/file.c
index 9a27336..9712bf7 100644
--- a/file.c
+++ b/file.c
@@ -2576,7 +2576,7 @@ utime_internal(const char *path, VALUE pathv, void *arg)
 #if defined(HAVE_UTIMENSAT)
     static int try_utimensat = 0;
 
-    if (try_utimensat) {
+/*    if (try_utimensat) {
         if (utimensat(AT_FDCWD, path, tsp, 0) < 0) {
             if (errno == ENOSYS) {
                 try_utimensat = 0;
@@ -2587,6 +2587,7 @@ utime_internal(const char *path, VALUE pathv, void *arg)
         return;
     }
 no_utimensat:
+*/
 #endif
 
     if (tsp) {
-- 
2.3.1


